app/src/main/java/com/example/resoluteapp/InboxFragment.java

```diff
@@ -40,56 +39,39 @@ public class InboxFragment extends Fragment {
 
     private FragmentInboxBinding binding;
 
-    FirebaseFirestore DB = FirebaseFirestore.getInstance();
+    private FirebaseFirestore DB = FirebaseFirestore.getInstance();
```

Comment:
I don't see how this change affects anything, but it's still confusing. Does making DB private allow something new to work?